stripe/res/values/strings.xml

```diff
@@ -4,48 +4,89 @@
+    <!--Label for input requesting the first line of an address, used for US and Canada addresses-->
     <string name="address_label_address">Address</string>
+    <!--Label for input requesting the first line of an address where input is optional, used for US and Canada addresses-->
     <string name="address_label_address_optional">Address (optional) </string>
+    <!--Label for input requesting the first line of an address, used for international addresses-->
     <string name="address_label_address_line1">Address line 1</string>
+    <!--Label for input requesting the first line of an address where , used for international addresses-->
     <string name="address_label_address_line1_optional">Address line 1 (optional)</string>
+    <!--Label for input requesting apartment number (address line 2), used for US and Canada based addresses-->
     <string name="address_label_apt">Apt. </string>
+    <!--Label for input requesting apartment number (address line 2) where input is optional, used for US based addresses-->
     <string name="address_label_apt_optional">Apt. (optional)</string>
+    <!--Label for input requesting address line 2, used for international addresses-->
     <string name="address_label_address_line2">Address line 2</string>
+    <!--Label for input requesting address line 2 where input in optional used for international addresses-->
     <string name="address_label_address_line2_optional">Address line 2 (optional)</string>
+    <!--Label for input requesting city, used for all locations-->
     <string name="address_label_city">City</string>
+    <!--Label for input requesting city where input is optional, used for all locations-->
     <string name="address_label_city_optional">City (optional)</string>
+    <!--Label for input requesting county, used for UK based addresses-->
     <string name="address_label_county">County</string>
+    <!--Label for input requesting county where input is optional , used for UK based addresses-->
     <string name="address_label_county_optional">County (optional)</string>
+    <!--Label for input requesting country, used for all locations-->
     <string name="address_label_country">Country</string>
-    <string name="address_label_country_optional">Country (optional)</string>
+    <!--Label for input requesting phone number, used for all locations-->
     <string name="address_label_phone_number">Phone number</string>
+    <!--Label for input requesting phone number where input is optional, used for all locations-->
     <string name="address_label_phone_number_optional">Phone number (optional)</string>
+    <!--Label for input requesting state, used for US based addreses-->
     <string name="address_label_state">State</string>
+    <!--Label for input requesting state where input is optional, used for US based addresses-->
     <string name="address_label_state_optional">State (optional)</string>
+    <!--Label for input requesting zip code, used for US based addreses-->
```

Comment:
Looks like later you go with "US addresses" instead of "US based addresses" (same with Canada just below this line). Probably should swap out the "based" forms, or else they should be hyphenated: ("US-based addresses")